.github/workflows/build-cmake-linux-armv7.yml

```diff
@@ -0,0 +1,58 @@
+name: armv7-cmake-dispatch
+
+on:
+  workflow_dispatch:
+    inputs:
+      cmake_version:
+        description: >
+          Cmake version to build and upload
+          For example 3.23.1
+        type: string
+        required: true
+
+jobs:
+  build-cmake:
+    name: Build cmake for linux-armv7
+    runs-on:
+      - self-hosted
+      - ARM
+    container:
+      image: ghcr.io/espressif/github-esp-dockerfiles/pyenv_rust_powershell:v1
+      options: --privileged
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Download and unpack cmake
+        shell: bash
```

Comment:
We don't need all of these"shell: bash" but we also have it in other workflows ...